packages/docs/docs/api/fhir/operations/evaluate-measure.md

```diff
@@ -0,0 +1,91 @@
+---
+sidebar_position: 20
+---
+
+# Measure $evaluate-measure
+
+Medplum supports the Measure [$evaluate-measure](https://hl7.org/fhir/R4/measure-operation-evaluate-measure.html) operation.
+
+:::caution
+
+`$evaluate-measure` is early and limited. At present, it only supports calculating counts using "application/x-fhir-query". Using FHIR `Library` is currently unsupported.
+
+:::
+
+## Usage
+
+First, create a FHIR `Measure`:
+
+```json
+{
```

Comment:
Can we move the examples to the `packages/examples` directory

Reply:
Hm, the advantage of the `examples` directory is type checking, and these are just JSON?